### PR TITLE
fix: TERM_PROGRAM is empty

### PR DESCRIPTION
**Pull request by z775729168**
TERM_PROGRAM is emtpy when you ssh remote hosts

### CODE REVIEW

**Comment by budimanjojo** [comment]
Sorry, but what is this PR for? The `$TERM_PROGRAM` was added by this PR: https://github.com/budimanjojo/tmux.fish/pull/1 to don't autostart `tmux` when it's inside VSCode.

**Comment by z775729168** [comment]
sorry for that.
if `$TERM_PROGRAM` is unset, `test ! $TERM_PROGRAM = 'vscode'` return false.
when I ssh a headless host, `$TERM_PROGRAM` is unset, so autostart failed.

**Comment by budimanjojo** [comment]
Thanks! I can confirm your problem. not sure if your patch will break the original PR. I'll try to find a better way to fix this.

File: conf.d/tmux.fish [hunk 1]
```diff
@@ -58,7 +58,7 @@ alias tmux=_fish_tmux_plugin_run
 
 set -q fish_tmux_autostarted || set fish_tmux_autostarted false
 if status is-interactive && ! fish_is_root_user
-    if test -z $TMUX && test $fish_tmux_autostart = true && test -z $INSIDE_EMACS && test -z $EMACS && test -z $VIM && test ! $TERM_PROGRAM = 'vscode'
+    if test -z $TMUX && test $fish_tmux_autostart = true && test -z $INSIDE_EMACS && test -z $EMACS && test -z $VIM && test ! "$TERM_PROGRAM" = 'vscode'
```
Comment (budimanjojo) [inline conf.d/tmux.fish]: can u change it to `test "$TERM_PROGRAM != "vscode"`?

**Review by budimanjojo — CHANGES_REQUESTED** [review]

File: conf.d/tmux.fish (hunk 1 above)
Comment (z775729168) [reply conf.d/tmux.fish]: I think it's OK.
Sorry, I never PR before, maybe I did something wrong.

**Review by budimanjojo — APPROVED** [review]

File: conf.d/tmux.fish (hunk 1 above)
Comment (budimanjojo) [reply conf.d/tmux.fish]: Nah there's nothing wrong here, it's easier to understand with "test this is not equal to that" than "test if not this equal to that" :D

**Comment by budimanjojo** [comment]
Thanks for the PR!

**Pull request merged**